.env

```diff
@@ -0,0 +1 @@
+COMPOSE_PROJECT_NAME=cantus
```

Comment:
not sure what you mean. what I was using that for is so that all the images have a constant name. images are named {PROJECT_NAME}_app etc. and at one point we need to reference one. This makes sure that reference is always valid, even if someone changes the name of the folder

Reply:
Gotcha. I was just asking because the name `.env` has a special meaning sometimes (e.g., python virtual environments) and wasn't sure if we needed that file committed as part of the repo. Based on what you said, we do need it.